docker-compose.yml

```diff
@@ -61,6 +62,18 @@ services:
       - database
     command: database:3306
 
+  skin_server_wmde19:
```

Comment:
Why does this need its own Docker service? Is this only for local development or would it also look like this on a production-like environment?

Reply:
The docker-compose.yml file is only for local development. I thought starting the dev server together with the rest would save us some trouble during development, because we don't have to remember (and document) starting another server. When the app is deployed, we compile the assets and then there is no need for the dev server.